### ISSUE TITLE
New problem to dock, Does not work at the end of the session

### ISSUE THREAD

**Issue by Leonardo425874** [micheleg/dash-to-dock#1658]
Good morning, I point out a new problem when I end the session the app dock disappears. Thank you very much for your work
![Schermata del 2022-01-25 20-47-45](https://user-images.githubusercontent.com/98007486/151052242-282b9187-9ae9-4801-bb44-1a2c63ce82b0.png)
.

**Comment by 3v1n0** [comment]
You mean while logging out?

It could be that gnome-shell disables all the extensions while it's shutting down. But I didn't check the code yet.

**Comment by Leonardo425874** [comment]
Yes, when I end the session and then I log back in, or when the computer goes into lock screen for inactivity and then I put the password back there is no dock. Thank you very much.
